test/PerfTests/JavaScript/ManyInstancesOrchestrator/index.js

```diff
@@ -0,0 +1,21 @@
+﻿/*
```

Comment:
I assume you named this "ManyInstancesOrchestrator" because that's the name of the test scenario it belongs to, but I suggest naming this function in a way that better reflects its implementation - e.g. "SequentialOrchestration" or something. "ManyInstances" doesn't reflect what this orchestrator does and can be confusing.